GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -581,6 +581,7 @@ void Renderer::renderRenderData(RenderData* render_data,
                             break;
                         case Material::ShaderType::EXTERNAL_RENDERER_SHADER:
                             shader_manager->getExternalRendererShader()->render(
+                                    mv_matrix, glm::inverseTranspose(mv_matrix),
```

Comment:
The parameters just mirror what is used by the texture shader. It's not my intention and one inversion save is not significant given the GHz speed of modern processor. We are bearing with some redundancy in the code as you can see.